components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -0,0 +1,120 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+
+#import "MDCFeatureHighlightAccessibilityMutator.h"
+#import "MaterialFeatureHighlight.h"
+
+#import <XCTest/XCTest.h>
+
+static NSArray<UIColor *> *testColors(){
+  return @[[UIColor whiteColor], [UIColor blackColor], [UIColor redColor], [UIColor orangeColor],
+           [UIColor greenColor], [UIColor blueColor], [UIColor grayColor]];
+}
+
+
+@interface FeatureHighlightTitleBodyAccessibilityMutatorTests : XCTestCase
+@property (nonatomic, strong) UIView *highlightedView;
+@property (nonatomic, strong) UIView *showView;
+@end
+
+@implementation FeatureHighlightTitleBodyAccessibilityMutatorTests
+
+- (void)setUp {
+  self.showView = [[UIView alloc] init];
+  self.highlightedView = [[UIView alloc] init];
+  [self.showView addSubview:self.highlightedView];
+}
+
+- (void)testMutatorChangesTextColor {
+  for (UIColor *color in testColors()) {
+    MDCFeatureHighlightViewController *hfvc =
+        [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:_highlightedView
```

Comment:
Nit: Please use property accessors instead of the backing ivars.

Reply:
done